src/components/header/Header.jsx

```diff
@@ -0,0 +1,39 @@
+import { CgProfile } from "assets/icons/icons";
+import { NavLink } from "react-router-dom";
+import { useState } from "react";
+
+const Header = () => {
+  const [isDisplay, setIsDisplay] = useState(false);
+
+  return (
+    <header className="sticky top-0 flex items-center relative w-full p-4 bg-green-600 text-slate-50">
+      <NavLink to="/">
+        <h1 className="text-4xl">Lucent Media</h1>
+      </NavLink>
+      <CgProfile
+        className="text-3xl ml-auto cursor-pointer"
+        onClick={() => setIsDisplay(!isDisplay)}
+      />
+      {isDisplay && (
```

Comment:
suggestion: can rename this variable to something more meaningful like showUserOptions...